### PR TITLE
docs: add docs for `useI18n` hook

### PR DESCRIPTION
**Pull request by dwelle**
https://docs-git-dwelle-docs-usei18n-excalidraw.vercel.app/docs/@excalidraw/excalidraw/api/utils#i18n

### CODE REVIEW

File: dev-docs/docs/@excalidraw/excalidraw/api/utils/utils-intro.md [hunk 1]
```diff
@@ -339,3 +339,46 @@ The `device` has the following `attributes`
...
+
+#### languages
+
+List of supported language codes. You can pass any of these to `Excalidraw`'s [`langCode` prop](/docs/@excalidraw/excalidraw/api/props/#langcode).
+
+#### useI18n
+
+A hook that returns the current language code and translation helper function. You can use this to translate strings in the components you render as children of `<Excalidraw>`.
+
+```jsx
```
Comment (ad1992) [inline dev-docs/docs/@excalidraw/excalidraw/api/utils/utils-intro.md:369]: live jsx ?

**Review by ad1992 — APPROVED** [review]
Made few tweaks and added live example.

File: dev-docs/package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     "@docusaurus/core": "2.2.0",
     "@docusaurus/preset-classic": "2.2.0",
     "@docusaurus/theme-live-codeblock": "2.2.0",
-    "@excalidraw/excalidraw": "next",
+    "@excalidraw/excalidraw": "0.14.2-e4d8ba2",
```
Comment (dwelle) [inline dev-docs/package.json:21]: what's this for

File: dev-docs/docs/@excalidraw/excalidraw/api/utils/utils-intro.md [hunk 3]
```diff
@@ -366,19 +366,18 @@ List of supported language codes. You can pass any of these to `Excalidraw`'s [`
 
 A hook that returns the current language code and translation helper function. You can use this to translate strings in the components you render as children of `<Excalidraw>`.
 
-```jsx
-import { Excalidraw, useI18n } from "@excalidraw/excalidraw";
```
Comment (dwelle) [inline dev-docs/docs/@excalidraw/excalidraw/api/utils/utils-intro.md]: isn't it better to be explicit?

**Pull request merged**